app/src/scripts/dataset/dataset.store.js

```diff
@@ -1363,14 +1347,10 @@ let datasetStore = Reflux.createStore({
         let datasetId = this.data.dataset.original
           ? this.data.dataset.original
           : this.data.dataset._id
-        router.transitionTo(
-          'snapshot',
-          {
-            datasetId: bids.decodeId(datasetId),
-            snapshotId: bids.decodeId(snapshotId),
-          },
-          { app: appLabel, version: appVersion, job: jobId },
-        )
+        const base = '/datasets/' + datasetId + '/versions/' + snapshotId
+        const query = { app: appLabel, version: appVersion, job: jobId }
+        const url = base + querystring(query)
+        history.push(url)
```

Comment:
I am getting an Uncaught TypeError here for history.push not being a function.  Looks like other methods have a history arg passed but dismissJobModal does not.

Reply:
I've pushed a fix for this.